source/libs/page-detect.ts

```diff
@@ -5,7 +5,7 @@ import select from 'select-dom';
 import {check as isReserved} from 'github-reserved-names';
 import {getUsername, getCleanPathname, getRepoPath, getOwnerAndRepo} from './utils';
 
-export const is404 = (): boolean => document.title === 'Page not found · GitHub';
+export const is404 = (): boolean => document.title === 'GitHub · Where software is built';
```

Comment:
🤷‍♂️

Reply:
Maybe we could use the alt text on the image? It contains `404`.

```
<img alt="404 “This is not the web page you are looking for”" class="js-plaxify position-absolute" data-xrange="20" data-yrange="10" height="249" width="271" style="z-index: 10; top: 72px; left: 72px; transform: translate3d(-6.09722px, -4.11647px, 0px);" src="data:image/png;base64,iVBOR...">
```

Reply:
I've submitted a pull request: #1849